bach/bach/describe.py

```diff
@@ -74,21 +55,72 @@ def __init__(
         if not self.df.data:
             raise ValueError('Cannot describe a Dataframe without columns')
 
-        self.include = _get_casted_filtering_dtypes(include)
-        self.exclude = _get_casted_filtering_dtypes(exclude)
+        self.series_to_describe = self.determine_series_to_describe(self.df, include, exclude)
 
         self.datetime_is_numeric = datetime_is_numeric
         self.percentiles = percentiles or [0.25, 0.5, 0.75]
 
         if self.percentiles and any(pt < 0 or pt > 1 for pt in self.percentiles):
             raise ValueError('percentiles should be between 0 and 1.')
 
+    @staticmethod
+    def determine_series_to_describe(
+            df: DataFrame,
+            include: Optional[Union[str, Sequence[str]]],
+            exclude: Optional[Sequence[str]]
+    ) -> List[str]:
```

Comment:
Just for keeping the same indentation and style in the module
```suggestion
    def determine_series_to_describe(
        df: DataFrame,
        include: Optional[Union[str, Sequence[str]]],
        exclude: Optional[Sequence[str]],
    ) -> List[str]:
```